components/DropdownLight/index.d.ts

```diff
@@ -14,6 +14,8 @@ export interface DropdownLightProps {
     spacing?: {};
     baseFontSize?: number;
   };
+  onChange?: (value: string) => void;
```

Comment:
Is the onChange argument always a string?? Check this as I believe this is wrong